### ISSUE TITLE
USWDS - Feature: Screen reader users lack context for breadcrumb links 

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#6114]
### Is your feature request related to a problem? Please describe.

Screen reader users who tab through breadcrumb links do not hear the "Breadcrumb" aria-label that's associated with the component wrapper. This can confuse users as it's lacking context of why these links are here.

### Describe the solution you'd like

Add "Breadcrumb link` aria-label to nested breadcrumb links

### Additional context

Discussed with a11y team in 10/15 sync. This needs usability research SPIKE to ensure this change will be welcome. 

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22) for duplicate feature requests.

**Comment by amyleadem** [comment]
## Next steps

We should answer:
- Do we know about this as an identified usability issue?
- Are there existing conventions for this change? Is there anything in the landscape that supports this change?

**Comment by amycole501** [comment]
Based on informal survey data from two screen reader users they confirmed that adding a name to the element is key to provide clarity on how/what the  user is supposed to do with that element.
quotes include:

_- I’ve found that more advanced users are familiar with these elements. Typically, your everyday user will be most familiar with the combo box. Please be sure to tag elements. Though it may not make sense to newer users of screen reading software, as you advance in your navigation skillset, you appreciate knowing the elements you are utilizing..
- In our basic AT training, we teach people to recognize and use common elements such as headings, links, combo boxes, radio buttons, check boxes, edit fields, tables, regions, lists, and buttons.  They learn the quick navigation keys to access each of those elements as well as the various ways to bring up an elements list._  

That said, "Breadcrumb" may not be the word we want to use, given it's using a visual cue to indicate what it is rather than a semantic one. We could call it instead "path navigation" or even "breadcrumb path navigation"

NOTE: All other design systems I researched did call this the "breadcrumb" or "breadcrumb trail".

**Comment by jaclinec** [comment]
Breadcrumb is slated for usability testing we plan to do in January 2025. Once we know more and have recommendations from the research, we can make more progress on this issue.

**Comment by dct2023** [comment]
@jaclinec Moving this one to 2025.4 as well.

**Comment by amycole501** [comment]
This issue was in the middle of undergoing UX research with disabled participants. Will review findings and update once the data is ready.
